dotcom-rendering/src/web/components/Branding.importable.tsx

```diff
@@ -1,46 +1,132 @@
 import { css } from '@emotion/react';
-import { neutral, textSans } from '@guardian/source-foundations';
+import { ArticleDesign } from '@guardian/libs';
+import { neutral, textSans, until } from '@guardian/source-foundations';
 import { trackSponsorLogoLinkClick } from '../browser/ga/ga';
 
 const brandingStyle = css`
 	padding-bottom: 10px;
 `;
 
-const brandingLabelStyle = css`
-	${textSans.xxsmall()};
-	color: ${neutral[20]};
-`;
+// for liveblog smaller breakpoints article meta is located in the same
+// container as standfirst and needs the same styling as standfirst
+function brandingLabelStyle(palette: Palette, format: ArticleFormat) {
```

Comment:
I've largely copied this from the existing implementation in `Contributor.tsx` https://github.com/guardian/dotcom-rendering/blob/8f393a42b9a01655dd95d7a492d51c191389cabe/dotcom-rendering/src/web/components/Contributor.tsx#L58-L85

I thought it best not to innovate too much, even though the use-case is a little different, but happy to refactor a bit if needed.